swagger-spec/comments/reports_list.yaml

```diff
@@ -1 +1,58 @@
 # /comments/{comment_id}/reports/
```

Comment:
Note to self: purposely not reviewing since this endpoint is commented out in swagger.yaml ATM.